spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_3_1_spec.rb

```diff
@@ -59,7 +59,7 @@ def assert_table_counts
     expect(ExtManagementSystem.count).to eq(1)
     expect(EmsFolder.count).to eq(7)
     expect(EmsCluster.count).to eq(4)
-    expect(Host.count).to eq(2)
+    expect(Host.count).to eq(1)
```

Comment:
@borod108 should this have changed because of a network factory?

Reply:
Ah, not sure why that would happen :/